.github/workflows/run_fvt_tests.yaml

```diff
@@ -201,12 +201,25 @@ jobs:
           echo "waiting emqx started";
           sleep 10;
         done
-    - name: Check cluster
+    - name: Get Token
       timeout-minutes: 10
       run: |
         kubectl port-forward service/${{ matrix.profile }} 18083:18083 > /dev/null &
+
+        while
```

Comment:
add a limit, e.g. 60 seconds.

Reply:
I have cutted down the limit from 10 minutes to 1.